src/mock/MockPolygonBridge.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import {IWorldIDIdentityManager} from "../interfaces/IWorldIDIdentityManager.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title Polygon WorldID Bridge Mock
+/// @author Worldcoin
+/// @dev of the Polygon FxPortal Bridge to test low-level assembly functions
+/// `grabSelector` and `stripSelector` in the PolygonWorldID contract
+contract MockPolygonBridge is Ownable {
```

Comment:
Inherit from the actual polygon bridge. You have _so much_ duplicated code.

Reply:
they are different, if I inherit I'll have to have lots of placeholder functions and I don't want that. I cleaned `MockPolygonBridge`. It's not that much duplicated code.